plasma/spec-tmpl.k

```diff
@@ -18,8 +18,8 @@
           <program> #asMapOpCodes(#dasmOpCodes(#parseByteStack({CODE}), BYZANTIUM)) </program>
           <programBytes> #parseByteStack({CODE}) </programBytes>
 
-          <id> ACCT_ID </id> // contract owner
-          <caller> CALLER_ID </caller> // who called this contract; in the begining, origin // msg.sender
+          <id> {OWNER} </id> // contract owner
```

Comment:
I think you don't need to put placeholder here in this case?

Reply:
yes, it should be removed (just trial and error)